dockerfiles/QAT/run.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+# NOTE: Some env variables are set in the Dockerfile - those that are
+# intended to be over-rideable.
+QAT_SRC=~/src/QAT
+export GOPATH=~/src/go
+export PATH=${PATH}:/usr/local/go/bin:${GOPATH}/bin
+
+packagerepo=github.com/kata-containers/packaging
+packagerepopath=${GOPATH}/src/${packagerepo}
+osbrepo=github.com/kata-containers/osbuilder
+osbrepopath=${GOPATH}/src/${osbrepo}
+testsrepo=github.com/kata-containers/tests
+testsrepopath=${GOPATH}/src/${testsrepo}
+runtimerepo=github.com/kata-containers/runtime
+runtimerepopath=${GOPATH}/src/${runtimerepo}
+agentrepo=github.com/kata-containers/agent
+agentrepopath=${GOPATH}/src/${agentrepo}
```

Comment:
This is a little hard to read - maybe space into blocks?

Reply:
Sure. had considered it. swings and roundabouts.